src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/domain/Oppfolgingstilfelle.kt

```diff
@@ -35,3 +36,6 @@ fun Oppfolgingstilfelle.toDialogmotekandidatStoppunktPlanlagt() =
         tilfelleStart = this.tilfelleStart,
         tilfelleEnd = this.tilfelleEnd,
     )
+
+fun List<Oppfolgingstilfelle>.onDate(date: LocalDate) =
```

Comment:
Lurt!

Reply:
Fint med en extension-funksjon her! 👍 Men jeg skjønner ikke helt navnet 🤔 `onDate` høres litt ut som boolean for meg, ikke at man skal returnere et tilfelle i listen som er på den datoen 🤷

Reply:
Ja, er litt usikker på navnet selv. `forDate` bedre kanskje? (vi bruker det i service-metoden som kaller denne). Åpen for forslag :)

Reply:
Kanskje `tilfelleForDate` eller noe, som spesifiserer at man kun returnerer ett av elementene i listen?

Reply:
Godt forslag, fikser!